src/core/server/server.api.md

```diff
@@ -839,7 +809,7 @@ export interface LegacyRequest extends Request {
 
 // @public @deprecated (undocumented)
 export interface LegacyServiceSetupDeps {
-    // Warning: (ae-incompatible-release-tags) The symbol "core" is marked as @public, but its signature references "InternalCoreSetup" which is marked as @internal
+    // Warning: (ae-forgotten-export) The symbol "InternalCoreSetup" needs to be exported by the entry point index.d.ts
     // 
     // (undocumented)
     core: InternalCoreSetup & {

```

Comment:
We still needs to export `LegacyServiceSetupDeps ` as it is used in `src/legacy/server/kbn_server.d.ts`, so the warning in documentation actually changes. Not sure we can do anything about this until legacy is dropped though?

Reply:
Yeah that's fine. Some of the doc warnings are ok if they're internal types.